distributed/protocol/serialize.py

```diff
@@ -132,8 +146,16 @@ def serialize(x, serializers=None, on_error="message", context=None):
     if isinstance(x, Serialized):
         return x.header, x.frames
 
+    # Check for "dask"-serializable data in dict/list/set
+    # Note: "msgpack" will always convert lists to tuples
+    #       list (see GitHub PR#2000), so we need to use
```

Comment:
Suggest fixing a small typo and referencing the issue for further discussion.

```suggestion
    #       (see GitHub #3716), so we need to use
```